nvflare/app_common/utils/fl_model_utils.py

```diff
@@ -19,6 +19,7 @@
 from nvflare.apis.fl_context import FLContext
 from nvflare.apis.shareable import Shareable
 from nvflare.app_common.abstract.fl_model import FLModel, FLModelConst, MetaKey, ParamsType
+from nvflare.app_common.abstract.model import ModelLearnable, ModelLearnableKey, make_model_learnable
```

Comment:
Not sure where should we put these conversion methods though.

Reply:
I think fl_model_utils makes sense since we also have the other conversion methods here as well. Do you have any other places in mind?